sdk/core/Azure.Core/api/Azure.Core.net5.0.cs

```diff
@@ -194,7 +194,9 @@ public partial class RequestContext
         public RequestContext() { }
         public System.Threading.CancellationToken CancellationToken { get { throw null; } set { } }
         public Azure.ErrorOptions ErrorOptions { get { throw null; } set { } }
+        public bool HasCustomClassifier { get { throw null; } }
         public void AddPolicy(Azure.Core.Pipeline.HttpPipelinePolicy policy, Azure.Core.HttpPipelinePosition position) { }
+        public void ConfigureResponse(int[] statusCodes, bool isError) { }
```

Comment:
This makes the classifier mutable. Does it mean it's now more dangerous to share it?

Reply:
Changed to a different approach.